### ISSUE TITLE
Back behavior blocked

### ISSUE THREAD

**Issue by Aldaviva** [Aldaviva/VivaldiCustomLauncher#9]
When the tweaks are applied to Vivaldi 6.4, the Back behavior of the keyboard shortcuts, gesture, quick command, and presumably the toolbar button does nothing. The Forward behavior still works. A workaround is to use the tab history menu `Alt`+`Z`.

Since it neither navigates back nor closes the tab, this probably means an uncaught exception is being thrown in the modified command handler function.

**Comment by Aldaviva** [comment]
In the console when running `vivaldi.exe --debug-packed-apps` and opening **Inspect portal-document**:
```text
Error in event handler: ReferenceError: vivaldiExports is not defined
    at Object.action [as function] (chrome-extension://mpognobbkildjkofajifpdfhcoklimli/bundle.js:1:3153251)
    at zhe (chrome-extension://mpognobbkildjkofajifpdfhcoklimli/bundle.js:1:2856584)
```

`vivaldiExports` is no more. I only found out about it very late, after it would have been helpful to have known about it much sooner, and it was removed very quickly after I started using it.

**Comment by Aldaviva** [comment]
Maybe this will do something. Reverting to calling the `canGoBack()` method on the `webview` element, which I used to do before portal elements were introduced and I couldn't get a reference to the webviews any more.
```js
document.getElementById("portals").__reactContainer$13rl6rc9epj.child.child.child.stateNode.state.appElm.querySelector(".webpageview.active webview").canGoBack()
```

In case the variable name changes:
```js
Object.entries(document.getElementById("portals")).find(b => b[0].startsWith("__reactContainer$"))[1].child /* ... */
```

Combined:
```js
Object.entries(document.getElementById("portals")).find(b => b[0].startsWith("__reactContainer$"))[1].child.child.child.stateNode.state.appElm.querySelector(".webpageview.active webview").canGoBack()
```

This call takes about 0.2 ms on my computer.

**Comment by Aldaviva** [comment]
Released in 1.2.13.
